### ISSUE TITLE
Post duplication issue on weak internet connections

### ISSUE THREAD

**Issue by sdubyam** [urbit/urbit#2145]
Hi there.  I'm away in Europe (my ship continues to run in San Francisco, and I have noticed that twice on relatively weak (2-4Mb/sec) internet connections I have posted once but seen a duplicate resultant post on the chat room.  One was a chat room I (somlus-savlev) hosted and one was hosted by minder-folden (/party).  

Nothing to do other than be aware that this issue exists and keep an eye out to see if it's a real problem for anyone else.  FYI the post was submitted only once (for sure) and ended up posting 46 times.  

https://i.imgur.com/9LxYAsj.png

**Comment by Fang-** [comment]
ime this is a visual issue with web chat, and the message itself doesn't actually get posted as many times as it's rendered. May or may not be a duplicate of #1985. Did others in the channel confirm they got the message 46 times, or was the issue only observed on your end?

**Comment by sdubyam** [comment]
Confirmed that the other participants in the chat saw the large multi-post in both chat rooms when it happened, so it appeared for everyone.

**Comment by Splined** [comment]
FWIW, I saw this for the somlus-savlev hosted one in the chat-cli as well 

https://imgur.com/a/dnY3lkq
